apps/web/src/components/PoolLeaderboard.tsx

```diff
@@ -19,7 +19,7 @@ export const Leaderboard = ({ address, token, donations = [] }: Props) => {
   );
   return (
     <section >
-      <h4 className="mb-2 text-xl font-semibold">Leaderboard</h4>
+      <h4 className={`${PPMori.className} mb-2 text-xl font-[500]`}>Leaderboard</h4>
```

Comment:
Here's another way of managing fonts in Next.js and Tailwind:
https://nextjs.org/docs/app/building-your-application/optimizing/fonts

That way you can set `font-sans` on the layout and class name `font-serif` on a headings component.